### PR TITLE
Stop requiring tls secrets after 2020.6.0

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

File: pkg/synopsysctl/cmd_create.go [hunk 1]
```diff
@@ -283,10 +283,23 @@ func checkPasswords(flagset *pflag.FlagSet) {
 	}
 }
 
-func checkSealKey(flagset *pflag.FlagSet) {
+func verifyUserProvidedSealKey(flagset *pflag.FlagSet) {
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_create.go]: why we need function for it?

Comment (mphammer) [reply pkg/synopsysctl/cmd_create.go]: I guess because it's used in 2 places (here and native)

Comment (mphammer) [reply pkg/synopsysctl/cmd_create.go]: @msenmurugan Or are you asking - why don't we put this logic into ctl_blackduck.go ?

Comment (msenmurugan) [reply pkg/synopsysctl/cmd_create.go]: we are not verifying anything in this method. so why do we need it?

Comment (mphammer) [reply pkg/synopsysctl/cmd_create.go]: We don't check the value but we are requiring that the user set --seal-key for the create command.

**Review by msenmurugan — APPROVED** [review]
/lgtm

**Pull request merged**